packages/hash/shared/src/prosemirror.ts

```diff
@@ -148,3 +259,65 @@ export const findComponentNode = (
 };
 
 export const componentNodeToId = (node: ComponentNode) => node.type.name;
+
+declare interface OrderedMapPrivateInterface<T> {
+  content: (string | T)[];
+}
+
+export const mutateSchema = (
```

Comment:
Can you explain what this is doing in a comment please?

Reply:
https://github.com/hashintel/hash/pull/872#:~:text=c92eceb-,Explain%20mutateSchema,-38cd5a7

Reply:
[`38cd5a7` (#872)](https://github.com/hashintel/hash/pull/872/commits/38cd5a7a13a94d48e3459218d6733485d9babe76)